src/radical/pilot/pmgr/launching/default.py

```diff
@@ -1119,6 +1121,23 @@ def _prepare_pilot(self, resource, rcfg, pilot, expand):
             if virtenv_mode != 'private':
                 cleanup = cleanup.replace('v', '')
 
+        # don't stage sdists on 'installed', 'local'
+        if virtenv_mode in ['installed', 'local']: stage_sdists = False
```

Comment:
`virtenv_mode == 'local' or rp_version == 'installed'`? there is no value `'installed'` for `virtenv_mode`

Reply:
Thanks, good catch!